fe/src/project/Seller.Portal/areas/Orders/components/EditOrderForm/EditOrderForm.js

```diff
@@ -111,7 +146,17 @@ function EditOrderForm(props) {
                             <a href={props.clientUrl}>{props.clientName}</a>
                         </div>
                     </div>
-                    
+                    <div className="mt-5">
```

Comment:
suggestion: Please add logic to show the button only for orders with an order state new. It makes no sense to provoke users with this button for orders that are already in progress, closed, etc.